ui/qt-lib/src/Ariadne/UI/Qt/Widgets/WalletInfo.hs

```diff
@@ -53,6 +54,9 @@ data WalletInfo =
     , itemModel :: QStandardItemModel.QStandardItemModel
     , selectionModel :: QItemSelectionModel.QItemSelectionModel
     , createAccountButton :: QPushButton.QPushButton
+    , currentItemType :: IORef (Maybe DeletingItem)
```

Comment:
Is it ok to use `DeletingItem` type here?

Reply:
I don't like it, defining a new data type seems better, because set of item types which can be deleted doesn't necessary have to be the same as set of item types which can be selected.